README.md

```diff
@@ -1 +1,341 @@
 placeholder
+
+# Chef Core
+[![Gem Version](https://badge.fury.io/rb/chef-chef-core.svg)](https://badge.fury.io/rb/chef-core)
+
+Chef Core provides low-level tools for building Chef workflows. It contains the subset of
```

Comment:
Workflows is such an overloaded word I worry about it here. That being said I don't have a great alternative.